keyboards/fractal/readme.md

```diff
@@ -0,0 +1,14 @@
+Fractal
+===
+
+The Fractal keyboard has a layout inspired by keyboards such as Preonic and JJ50. It aims to deliver a good quality but affordable keyboard. It supports Alps and MX switches and uses a Pro Micro as the controller.
+
+Keyboard Maintainer: The QMK Community
```

Comment:
Make sure that these have a double space at the end of each of the hardware lines, so that it's properly formatted.